src/types/index.d.ts

```diff
@@ -110,14 +110,33 @@ export interface TokenOverlay {
   mod: TokenMod
 }
 
+export type FileBuffer = number[]
+export interface oFile {
```

Comment:
This can be the native `File` type, no? Interface will remain the same

Reply:
done